src/test/java/tests/utilities/SettingsFileTests.java

```diff
@@ -0,0 +1,121 @@
+package tests.utilities;
+
+import aquality.selenium.core.utilities.ISettingsFile;
+import com.google.common.base.Charsets;
+import com.google.common.io.Files;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import tests.application.CustomAqualityServices;
+import tests.application.TestModule;
+import java.io.File;
+import java.io.IOException;
+import java.util.*;
+
+import static org.testng.Assert.*;
+
+public class SettingsFileTests {
+    private static final String timeoutPollingIntervalPath = "/timeouts/timeoutPollingInterval";
```

Comment:
Please change names of all of static fields to upper-case format, according to Java code conventions 

> The names of variables declared class constants and of ANSI constants should be all uppercase with words separated by underscores ("_"). (ANSI constants should be avoided, for ease of debugging.)